packages/opentelemetry/README.md

```diff
@@ -4,4 +4,181 @@
 An [OpenTelemetry](https://opentelemetry.io/docs/what-is-opentelemetry/) client that's preconfigured for drop-in use in FT apps. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
 
 > [!WARNING]<br />
-> This package is in beta and hasn't been tested extensively in production yet. Feel free to use, and any feedback is greatly appreciated.
\ No newline at end of file
+> This package is in beta and hasn't been tested extensively in production yet. Feel free to use, and any feedback is greatly appreciated.
+
+* [Usage](#usage)
+  * [Automated setup with `--require`](#automated-setup-with---require)
+  * [Automated setup with `require()`](#automated-setup-with-require)
+  * [Manual setup](#manual-setup)
+  * [Configuration options](#configuration-options)
+    * [`options.authorizationHeader`](#optionsauthorizationheader)
+    * [`options.tracesEndpoint`](#optionstracesendpoint)
+    * [`OTEL_` environment variables](#otel_-environment-variables)
+* [Contributing](#contributing)
+* [License](#license)
+
+
+## Usage
+
+Install `@dotcom-reliability-kit/opentelemetry` as a dependency:
+
+```bash
+npm install --save @dotcom-reliability-kit/opentelemetry
```

Comment:
non-blocking: we only want it as a dev dependency? We don't want it to have access to it when running in production?

Reply:
Nope we want it as a prod dependency which `--save` does 🙂